lib/xgu/nv2a_regs.h

```diff
@@ -0,0 +1,1321 @@
+#       define NV097_SET_SKIN_MODE_4G                             5
+#       define NV097_SET_SKIN_MODE_4                              6
+#   define NV097_SET_STENCIL_TEST_ENABLE                      0x0000032C
+#   define NV097_SET_POLY_OFFSET_POINT_ENABLE                 0x00000330
+#   define NV097_SET_POLY_OFFSET_LINE_ENABLE                  0x00000334
+#   define NV097_SET_POLY_OFFSET_FILL_ENABLE                  0x00000338
+#   define NV097_SET_ALPHA_FUNC                               0x0000033C
+#   define NV097_SET_ALPHA_REF                                0x00000340
+#   define NV097_SET_BLEND_FUNC_SFACTOR                       0x00000344
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ZERO                0x0000
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE                 0x0001
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_SRC_COLOR           0x0300
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_SRC_COLOR 0x0301
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_SRC_ALPHA           0x0302
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_SRC_ALPHA 0x0303
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_DST_ALPHA           0x0304
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_DST_ALPHA 0x0305
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_DST_COLOR           0x0306
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_DST_COLOR 0x0307
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_SRC_ALPHA_SATURATE  0x0308
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_CONSTANT_COLOR      0x8001
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_CONSTANT_COLOR 0x8002
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_CONSTANT_ALPHA      0x8003
+#       define NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_CONSTANT_ALPHA 0x8004
+#   define NV097_SET_BLEND_FUNC_DFACTOR                       0x00000348
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ZERO                0x0000
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE                 0x0001
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_SRC_COLOR           0x0300
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE_MINUS_SRC_COLOR 0x0301
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_SRC_ALPHA           0x0302
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE_MINUS_SRC_ALPHA 0x0303
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_DST_ALPHA           0x0304
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE_MINUS_DST_ALPHA 0x0305
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_DST_COLOR           0x0306
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE_MINUS_DST_COLOR 0x0307
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_SRC_ALPHA_SATURATE  0x0308
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_CONSTANT_COLOR      0x8001
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE_MINUS_CONSTANT_COLOR 0x8002
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_CONSTANT_ALPHA      0x8003
+#       define NV097_SET_BLEND_FUNC_DFACTOR_V_ONE_MINUS_CONSTANT_ALPHA 0x8004
```

Comment:
Enum for `NV097_SET_BLEND_FUNC_DFACTOR`; possibly overlap with `NV097_SET_BLEND_FUNC_SFACTOR`, so if name and value match, consider merging them.

Reply:
They are a match. Added a comment to xgu code.